### ISSUE TITLE
Step #5: error: /workspace/out/libfuzzer-coverage-x86_64/dumps/*.profdata: No such file or directory

### ISSUE THREAD

**Issue by Jens-G** [google/oss-fuzz#7208]
https://oss-fuzz-build-logs.storage.googleapis.com/log-20a7836e-a8d2-4924-9ee6-67aef2c027cb.txt

Not much I can do here. Logged since April 2021

**Comment by DavidKorczynski** [comment]
Thanks, I believe https://github.com/google/oss-fuzz/pull/7199 is meant to address this one

**Comment by jonathanmetzman** [comment]
@DavidKorczynski I don't think this is affected by #7199, particularly if it has been going on since April 2021 as @Jens-G says.
Am I missing something in the logs?

**Comment by DavidKorczynski** [comment]
I think you're right. Am not entirely sure why I thought your PR affected this one - looking at the log now it doesn't look to be affected by fuzz-introspector at all

**Comment by DavidKorczynski** [comment]
Is this a coverage problem or something with the build? I see these lines:

```
Step #3 - "compile-libfuzzer-coverage-x86_64": + go mod tidy
Step #3 - "compile-libfuzzer-coverage-x86_64": + compile_go_fuzzer . Fuzz fuzz_go_tutorial
Step #3 - "compile-libfuzzer-coverage-x86_64": /usr/local/bin/compile_go_fuzzer: line 28: cd: /root/go/src/.: No such file or directory
Step #3 - "compile-libfuzzer-coverage-x86_64": github.com/apache/thrift/lib/go/test/fuzz
Step #3 - "compile-libfuzzer-coverage-x86_64": github.com/apache/thrift/lib/go/test/fuzz
```

Which seems to indicate something (not `*profdata`) is missing?

**Comment by DavidKorczynski** [comment]
Perhaps because it inherits two base builders? 
https://github.com/google/oss-fuzz/blob/898bbe41e57f841d15cb9d30bd42105460857386/projects/thrift/Dockerfile#L17-L20
